src/stdfblib/events/GEN_E_MUX_fbt.cpp

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2011, 2023 ACIN, Profactor GmbH, fortiss GmbH
```

Comment:
You forgot the year!

Reply:
you are right. 
will do Friday, Embedded World tomorrow, Excavator driving Thursday.